src/app/templates/app/license_requests.html

```diff
@@ -70,14 +73,14 @@
     });
     $('body').on('click', 'button[id^="archive_button"]', function (en) {
       en.stopImmediatePropagation();
-      $("#modal-header").addClass("yellow-modal");
-	    $(".modal-footer").html('<button class="btn btn-success" id="confirm_archive"><span class="glyphicon glyphicon-off"></span> Confirm</button>');
-      $("#modal-body").html('Are you sure you want to archive this license?');
-      $("#myModal").modal({
-        backdrop: 'static',
-        keyboard: true,
-        show: true,
-      })
+       $("#modal-header").addClass("yellow-modal");
+       $("#modal-body").html('Are you sure you want to archive this license?');
+       $("#myModal").modal({
+                backdrop: 'static',
+                keyboard: true,
+                show: true,
+          })
+      
```

Comment:
<img width="599" alt="Screenshot 2020-04-28 at 6 38 44 PM" src="https://user-images.githubusercontent.com/17159160/80490820-8ac15780-897f-11ea-974f-19028d011533.png">
Dialog box does not have submit button.
Please test all things before commiting.